src/cmap/connection_pool.ts

```diff
@@ -355,6 +356,7 @@ export class ConnectionPool extends TypedEventEmitter<ConnectionPoolEvents> {
    * explicitly destroyed by the new owner.
    */
   async checkOut(): Promise<Connection> {
+    const checkoutTime = Date.now();
```

Comment:
Instead of Date.now(), we usually use the `now()` utility (it's in `utils.ts`).  Can we adjust this PR to use `now` too?

Reply:
yep!